src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -59,6 +59,13 @@
 			class="tw-mb-1.5" :style="{width: '100%', height: '5.5rem'}"
 		/>
 
+		<div
+			v-if="!isLoading"
+			class="tw-mb-1.5 lg:tw-mb-1"
+		>
+			{{ loanUse }}
```

Comment:
I think we need to determine the character count restriction for this field, then truncate and add an ellipsis if we cut the content at that limit.

Reply:
Will add that and have it up shortly.